theme/layouts/homepage.ejs

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <% include _head %>
+  </head>
+  <body class="homepage">
+    <div>
+      <header>
+        <h1>
+          <a href="\">
+            <%= site.title %>
+          </a>
+        </h1>
+        <p>
+          <%= site.subtitle %>
+        </p>
+        <% include navigation/_main %>
+      </header>
```

Comment:
Suggestion/separation: What about moving this into `_header.ejs`, this way we will have less duplications in our layouts and it will be consistent with other separations we had.